File: URI_1008_salary.js
const lines = ["25", "100", "5.50"];

let emplNumber = lines[0];
let workedHoursPerMonth = lines[1];
let salaryPerHour = lines[2];

let salaryPerMonth = salaryPerHour * workedHoursPerMonth;

console.log("NUMBER = " + emplNumber);
console.log("SALARY = U$ " + salaryPerMonth.toFixed(2));

File: URI_1008_salary.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class URI_1008_salary {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		
		int emplNumber = scanner.nextInt();
		int workedHoursPerMonth = scanner.nextInt();
		float salaryPerHour  = scanner.nextFloat();
		
		float salaryPerMonth = salaryPerHour * workedHoursPerMonth;
		String formattedSalaryPerMonth = new DecimalFormat("0.00").format(salaryPerMonth);
		
		System.out.println("NUMBER = " + emplNumber);
		System.out.println("SALARY = U$ " + formattedSalaryPerMonth);

		scanner.close();
	}
}
